.github/workflows/build.yml

```diff
@@ -0,0 +1,21 @@
+name: Node Build
+ 
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+ 
+jobs:
+  build:
+    runs-on: ubuntu-latest
+ 
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
why checkout@v3?

Reply:
Not sure- I made some changes based on the doc you sent and this is now @v4